frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -444,6 +429,16 @@ export const CreateVMWizardPageComponent: React.FC<CreateVMWizardPageComponentPr
 
   const storageClassConfigMap = useStorageClassConfigMapWrapped();
 
+  const loadedCommonTemplates = React.useMemo(
+    () =>
+      userMode === VMWizardMode.IMPORT // make empty for import so no PVCBaseImages get loaded
+        ? []
+        : immutableListToShallowJS(iGetLoadedData(commonTemplates)),
+    [commonTemplates, userMode],
+  );
+  // TODO integrate the list of watches into the redux store to prevent unnecessary copying of data
+  const openshiftCNVBaseImages = useWatchedResultAsObject(usePVCBaseImages(loadedCommonTemplates));
```

Comment:
`usePVCBaseImages` already memoizes the output object

Reply:
ah, this is used as converter to object. But why hook form ?

Reply:
because they use `useMemo` hook inside and thus should be called as a hook unconditionally.

Reply:
sure, but why are you creating a hook which only wraps `useMemo` ? Just use `useMemo` directly.

Reply:
this is just to simplify things and have one call which does what is required. Having two calls beets the point of extracting this functionality a bit IMO. 

We can also inline the functionality here, if you think such small optimization doesn't make sense.

Reply:
> this is just to simplify things 

No, it is adding cyclomatic complexity (because we use useMemo twice) and (119-65) code lines and one file.
It is also harder for a newcomers to the code because they will have one more object to learn.

Reply:
I would prefer inlining :+1:

Reply:
> No, it is adding cyclomatic complexity (because we use useMemo twice) and (119-65) code lines and one file.

we need to use `useMemo` twice either way, so the cyclomatic complexity will not change in total, unless I modify the `usePVCBaseImages` function.

> It is also harder for a newcomers to the code because they will have one more object to learn.

this is not so easy topic and for longer discussion :)

anyway, this was just a small refactoring - np, we can inline it

Reply:
> this is not so easy topic and for longer discussion :)

:smile: